docs/core/tools/dotnet-new.md

```diff
@@ -33,28 +32,28 @@ After this, the project is ready to be compiled and/or edited further.
 
 ## Options
 
-`-l`, `--lang [C#|F#]`
+`-h|--help`
 
-Language of the project. Defaults to `C#`. `csharp` (`fsharp`) or `cs` (`fs`) are also valid options.
+Prints out a short help for the command.  
+
+`-l|--lang <C#|F#>`
+
+Language of the project. Defaults to `C#`. Valid values are `csharp`, `fsharp`, `cs` and `fs`.
 
 `-t`, `--type`
 
-Type of the project. Valid values are `console`, `web`, `lib` and `xunittest`. 
+Type of the project. Valid values for C# are `console`, `web`, `lib` and `xunittest` and for F# only `console` is valid. 
 
-## EXAMPLES
+## Examples
 
-`dotnet new`
-    
-Drops a C# project in the current directory.
+Create a C# project in the current directory:
 
-`dotnet new --lang f#`
-    
-Drops an F# project in the current directory.
+`dotnet new` or `dotnet new --lang c#` 
+   
+Create an F# project in the current directory:
 
-`dotnet new --lang c#`
-    
-Drops an C# project in the current directory.
+`dotnet new --lang f#`
+  
+Create a new ASP.NET Core project in the current directory:
 
-`dotnet new -t web`
-    
-Drops a new ASP.NET Core project in the current directory. 
+`dotnet new -t web`
```

Comment:
Should these descriptions be more explicit about the defaults? I.e. say that the first two examples create a console application and the the third example creates a C# application.